session.go

```diff
@@ -203,174 +196,13 @@ type RESTChannel interface {
 	KickParticipant(ctx context.Context, channelID, userID Snowflake, flags ...Flag) (err error)
 }
 
-// RESTEmoji REST interface for all emoji endpoints
-type RESTEmoji interface {
-	// GetGuildEmoji Returns an emoji object for the given guild and emoji IDs.
-	GetGuildEmoji(ctx context.Context, guildID, emojiID Snowflake, flags ...Flag) (*Emoji, error)
-
-	// GetGuildEmojis Returns a list of emoji objects for the given guild.
-	GetGuildEmojis(ctx context.Context, id Snowflake, flags ...Flag) ([]*Emoji, error)
-
-	// CreateGuildEmoji Create a new emoji for the guild. Requires the 'MANAGE_EMOJIS' permission.
-	// Returns the new emoji object on success. Fires a Guild Emojis Update Gateway event.
-	CreateGuildEmoji(ctx context.Context, guildID Snowflake, params *CreateGuildEmojiParams, flags ...Flag) (*Emoji, error)
-
-	// UpdateGuildEmoji Modify the given emoji. Requires the 'MANAGE_EMOJIS' permission.
-	// Returns the updated emoji object on success. Fires a Guild Emojis Update Gateway event.
-	UpdateGuildEmoji(ctx context.Context, guildID, emojiID Snowflake, flags ...Flag) *updateGuildEmojiBuilder
-
-	// DeleteGuildEmoji Delete the given emoji. Requires the 'MANAGE_EMOJIS' permission. Returns 204 No Content on
-	// success. Fires a Guild Emojis Update Gateway event.
-	DeleteGuildEmoji(ctx context.Context, guildID, emojiID Snowflake, flags ...Flag) error
-}
-
 // RESTGuild REST interface for all guild endpoints
 type RESTGuild interface {
 	// CreateGuild Create a new guild. Returns a guild object on success. Fires a Guild Create Gateway event.
 	CreateGuild(ctx context.Context, guildName string, params *CreateGuildParams, flags ...Flag) (*Guild, error)
```

Comment:
Any reason you did not move the create method into the guild resource?

Reply:
Yeah, that was done because .Guild needs an ID which we do not have if the guild doesn't exist.